compiler/luci-interpreter/src/kernels/Mul.cpp

```diff
@@ -36,7 +36,13 @@ Mul::Mul(const Tensor *input1, const Tensor *input2, Tensor *output, const MulPa
 
 void Mul::configure()
 {
-  assert(input1()->element_type() == input2()->element_type());
+  LUCI_INTERPRETER_CHECK(input1()->element_type() == input2()->element_type());
+  if (input1()->element_type() == DataType::S16)
+  {
+    LUCI_INTERPRETER_CHECK(input1()->zero_point() == 0 && input2()->zero_point() == 0 &&
```

Comment:
Should output quantization be same as input? I mean is operators (S16, S16) -> U8 valid?
Or this is checked somewhere else?

Reply:
No, I'll add the check.

Reply:
Fixed.